espresso-server/app/build.gradle

```diff
@@ -14,8 +16,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            minifyEnabled true
```

Comment:
why should we enable that? this setting might corrupt class names, so they will not be visible for reflection calls